docs/devices/9290030674.md

```diff
@@ -34,6 +34,8 @@ Please note that the setup light is below the main sensor
 
 This specific device has been reported to have issues repairing to a Zigbee network after upgrading from a CC2531 to a CC2652 controller (Zigbee 1.2 to 3.0). (Re)pairing may only work after pairing the device to another network and channel first (has been tested with a Philips Hue 2.0 hub in this instance) before pairing it back to the Zigbee2MQTT network again.
 
+Using Conbee II there are reported issues with re-pairing after a power outage.  This may be overcome by adding the device to the 'passlist' and setting permit_join for the network to true in your config file.
```

Comment:
`passlist` shouldn't do anything, it's only the permit joining probably

Reply:
Sure, yes.  Was trying to encourage open networks to use passlists.  Can either make that more explicit or remove the reference.

Reply:
The passlist does not really prevent devices from joining, so it doesn't make stuff any safer. Please remove it.